test/helpers/kubectl.go

```diff
@@ -117,6 +117,31 @@ func (kub *Kubectl) CepGet(namespace string, pod string) *models.Endpoint {
 	return data
 }
 
+// WaitForEnforcingCNP waits for the CNP with the specified to go into
+// enforcing mode on all nodes
+func (kub *Kubectl) WaitForEnforcingCNP(cnpName string) error {
+	body := func() bool {
+		for _, node := range []string{K8s1, K8s2} {
+			cnpCMD := fmt.Sprintf("%s get cnp %s -o jsonpath='{range .status.nodes.%s}{.enforcing}{end}'", KubectlCmd, cnpName, node)
```

Comment:
This should get a namespace, multiple test are using a second namespace.

Reply:
Why not range over nodes and get only one time and validate that all are enforcing?